src/Network/StatsD/Datadog.hs

```diff
@@ -395,18 +398,83 @@ data StatsClient = StatsClient
 -- >   send client $ event "Wombat attack" "A host of mighty wombats has breached the gates"
 -- >   send client $ metric "wombat.force_count" Gauge (9001 :: Int)
 -- >   send client $ serviceCheck "Wombat Radar" ServiceOk
-send :: (MonadBase IO m, ToStatsD v) => StatsClient -> v -> m ()
-send (StatsClient _ r) v = liftBase $ reaperAdd r (toStatsD v >> appendChar7 '\n')
-send Dummy _ = return ()
+-- send :: (MonadBase IO m, ToStatsD v) => StatsClient -> v -> m ()
```

Comment:
Yes, eventually I will perhaps. There was code commented before me, so I lost track which one was mine and which one was there